resources/log4j.properties

```diff
@@ -20,5 +20,6 @@ log4j.appender.metabase=metabase.logger.Appender
 log4j.logger.metabase=INFO
 log4j.logger.metabase.db=DEBUG
 log4j.logger.metabase.driver=DEBUG
+log4j.logger.metabase.middleware=DEBUG
```

Comment:
logging everything that happens in the app is overly verbose and unnecessary.  It makes it difficult to use the logs for finding relevant information about what's important to tell the user and increases the likelihood that true errors are missed.